frontend/src/components/task/task-explorer.js

```diff
@@ -121,6 +121,20 @@ class TaskExplorer extends Component {
     }
   }
 
+  componentWillReceiveProps (props) {
+    switch (props.history.location.pathname) {
+      case '/tasks/open':
+        this.setState({ value: 0, showNavigation: true })
+        break
+      case '/projects':
+        this.setState({ value: 1, showNavigation: true })
+        break
+      case '/organizations':
+        this.setState({ value: 2, showNavigation: true })
+        break
+    }
```

Comment:
@VictorAlessander `switch` should have a default case here, I don't why this passes on lin :p

Maybe a default case that it will do nothing, just with the break

Reply:
Inserted a default condition with just the break.